floor/README.md

```diff
@@ -395,7 +395,7 @@ class Person {
 ```
 
 ### Ignoring Fields
-The `hashCode` property and all static fields of entities are ignored by default and thus excluded from the library's mapping.
+Getters, setters, operators and all static fields of entities are ignored by default and thus excluded from the library's mapping.
```

Comment:
❓What do you mean by "operators" here?

Reply:
the overridable operators like ==, +, etc.. but this is probably as obvious as other functions... I should remove that.